payloads/library/general/FICBunny/readme.md

```diff
@@ -0,0 +1,38 @@
+## FICBunny
+
+* Title: FICBunny
+* Short Description: Firmware Image Creator for the Bash Bunny
+* Author: HSF3232 (@Hacksawfred3232)
+* Contributer: Austin Spraggins (@spragginsdesigns)
+* Version: 1.0
+* Last tested Bunny Firmware version: 1.7
+
+## Long Description
+
+The primary purpose of this script is to create a backup image (in case you want to revert to a known good point). Then replace the missing firmware image within /dev/nandg, should it be missing.
+
+Note: It may be a good idea to disable all any non-critical services - if any - on the bunny before starting this payload. 

```

Comment:
It may be a good idea to disable any non-critical services - if any - on the bunny before starting this payload.
- Quick Grammar Suggestion

Reply:
Fixed it. Thank you for pointing it out.

-- HSF3232